Example/Backpack/NavigationData.swift

```diff
@@ -0,0 +1,247 @@
+        Item(name: "Elevated", result: Storyboard(storyboardName: "Panel", storyboardIdentifier: "PanelViewController"))
+    ])]))
+    static var progressItem = Item(name: "Progress bar", result: Storyboard(storyboardName: "ProgressBar", storyboardIdentifier: "ProgressBarViewController"))
+    static var ratingItem = Item(name: "Rating", result: Storyboard(storyboardName: "Ratings", storyboardIdentifier: "RatingsViewController"))
+    static var snackbarItem = Item(name: "Snackbar", result: AppStructure(sections: [Section(name: "", items: [
+        Item(name: "With text", result: Storyboard(storyboardName: "Snackbar", storyboardIdentifier: "SnackbarViewController")),
+        Item(name: "With text and buttons", result: Storyboard(storyboardName: "Snackbar", storyboardIdentifier: "SnackbarViewController")),
+        Item(name: "With title, text and button", result: Storyboard(storyboardName: "Snackbar", storyboardIdentifier: "SnackbarViewController")),
+        Item(name: "With text and icon only button", result: Storyboard(storyboardName: "Snackbar", storyboardIdentifier: "SnackbarViewController")),
+        Item(name: "With text, button and accessory icon", result: Storyboard(storyboardName: "Snackbar", storyboardIdentifier: "SnackbarViewController")),
+        Item(name: "With indefinite duration", result: Storyboard(storyboardName: "Snackbar", storyboardIdentifier: "SnackbarViewController")),
+        Item(name: "With long duration", result: Storyboard(storyboardName: "Snackbar", storyboardIdentifier: "SnackbarViewController")),
+        Item(name: "With short duration", result: Storyboard(storyboardName: "Snackbar", storyboardIdentifier: "SnackbarViewController")),
+        Item(name: "With keyboard", result: Storyboard(storyboardName: "Snackbar", storyboardIdentifier: "SnackbarViewController")),
+        Item(name: "With delegate", result: Storyboard(storyboardName: "Snackbar", storyboardIdentifier: "SnackbarViewController"))
+    ])]))
+    static var spinnersItem = Item(name: "Spinners", result: Storyboard(storyboardName: "Spinners", storyboardIdentifier: "SpinnersViewController"))
+    static var starItem = Item(name: "Star ratings", result: AppStructure(sections: [Section(name: "", items: [
+        Item(name: "Interactive", result: Storyboard(storyboardName: "StarRatings", storyboardIdentifier: "StarRatingsViewController")),
+        Item(name: "Docs", result: Storyboard(storyboardName: "StarRatings", storyboardIdentifier: "StarRatingsViewController"))
+    ])]))
+    static var switchesItem = Item(name: "Switches", result: Storyboard(storyboardName: "Switches", storyboardIdentifier: "SwitchesViewController"))
+    static var tabItem = Item(name: "Tab bar controller", result: Storyboard(storyboardName: "TabBarControllers", storyboardIdentifier: "TabBarControllerViewController"))
+    static var tappableItem = Item(name: "Tappable link labels", result: AppStructure(sections: [Section(name: "", items: [
```

Comment:
If the `name` of  a `Section` is optional isn't this one of the cases where we'd want to use that over an empty string?